openwisp_notifications/templates/openwisp_notifications/preferences.html

```diff
@@ -0,0 +1,57 @@
+{% extends "admin/base_site.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block title %}
+  {% trans "Notification Preferences" %}
+{% endblock title %}
+
+{% block extrastyle %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static 'openwisp-notifications/css/preferences.css' %}" />
+{% endblock extrastyle %}
+
+{% block breadcrumbs %}
+  <div class="breadcrumbs">
+    <a href="{% url 'admin:index' %}">{% trans 'Home' %}</a> &rsaquo;
+    {% if username %}
+      <a href="{% url 'admin:app_list' app_label='openwisp_users' %}">{% trans 'Users and Organizations' %}</a> &rsaquo;
+      <a href="{% url 'admin:openwisp_users_user_changelist' %}">{% trans 'Users' %}</a> &rsaquo;
+      <a href="{% url 'admin:openwisp_users_user_change' user_id %}">{{ username }}</a> &rsaquo;
+    {% endif %}
+    {% trans 'Notification Preferences' %}
+  </div>
+{% endblock breadcrumbs %}
+
+{% block content %}
+  <div class="settings-container" data-user-id="{{ user.id }}">
```

Comment:
This is reading the logged in user's ID.

Reply:
```suggestion
  <div class="settings-container" data-user-id="{{ user_id }}">
```

Add a failing test for this.

Reply:
Sorry about that mistake, I'll be more careful next time!

Reply:
Also it's been handled here 

https://github.com/openwisp/openwisp-notifications/blob/7982603fe9e51531b1e12b82318b0eb377bbced8/openwisp_notifications/tests/test_notifications.py#L1074-L1077